app/controllers/integrations_controller.rb

```diff
@@ -0,0 +1,9 @@
+class IntegrationsController < ApplicationController
+  active_tab "integrations"
+
+  require_permission :can_view_integrations?
+
+  def show
+    @total_inventory_value = Category.all.to_a.sum(&:value)
```

Comment:
This will be more efficient if you do `Category.includes(:items).all.to_a.sum(&:value)` since `value` references the `items` association.

Reply:
NOTE: I'm not sure if `.all` when you do `.includes` so you may want to test this out of course.